### PR TITLE
Added hrkrshnn and axic to contributors.md

### PR DESCRIPTION
**Pull request by hrkrshnn**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -56,3 +56,4 @@ dmfxyz                         | `dmfxyz.eth`
 daltoncoder                    | `dontkillrobots.eth`
 0xf4ce                         | `0xf4ce.eth`
 phaze                          | `phaze.eth`
+Hari                           | `hrkrshnn.eth`
```
Comment (leonardoalt) [inline CONTRIBUTORS.md:59]: Shouldn't the LHS be the GH handle?

Comment (hrkrshnn) [reply CONTRIBUTORS.md:59]: ah, nice. Will fix.

Comment (hrkrshnn) [reply CONTRIBUTORS.md:59]: Hmm, there are names and also handles. Will just use handle for now.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -56,3 +56,5 @@ dmfxyz                         | `dmfxyz.eth`
 daltoncoder                    | `dontkillrobots.eth`
 0xf4ce                         | `0xf4ce.eth`
 phaze                          | `phaze.eth`
+hrkrshnn                       | `hrkrshnn.eth`
+axic                           | `axic.eth`
```
Comment (hrkrshnn) [inline CONTRIBUTORS.md:60]: Was added from request by @axic. Just to minimize PRs.

**Pull request merged**